readme.md

```diff
@@ -110,6 +110,7 @@ Click the type names for complete docs.
 
 - [`EmptyObject`](source/empty-object.d.ts) - Represents a strictly empty plain object, the `{}` value.
 - [`IsEmptyObject`](source/empty-object.d.ts) - Returns a `boolean` for whether the type is strictly equal to an empty plain object, the `{}` value.
+- [`UnknownRecord`](source/unknown-record.d.ts) - Matches any unknown record.
```

Comment:
I copied this description from the internal jsDoc. Is it good enough?

Reply:
Maybe:

> Represents an object with any value. You probably want this instead of `{}`.

Reply:
I'm not sure it's better, the word `any` might confuse here. WDYT about "Represents an object with unknown keys and values"?

Reply:
@MatanYadaev Your version can be confusing too as the keys are not `unknown`.

I would go with:

> Represents an object with `unknown` value. You probably want this instead of `{}`.